tr.yml

```diff
@@ -0,0 +1,279 @@
+      noData: "Mod kaynağı eksik."
+      collapseExtra: "Beni Oku ve Ayarları Daralt"
+    changelog:
+      title: "Değişiklikler"
+      loadingFailed: "Değişiklikler yüklenemedi. GitHub'da görüntülemek için <0>burayı tıklayın</0>."
+    advanced:
+      title: "Gelişmiş"
+      debugLogging:
+        title: "Hata ayıklama günlüğü"
+        description: "Modla ilgili sorunları gidermek için yararlı olabilir."
+        none: "Hiçbiri"
+        modLogs: "Mod günlükleri"
+        detailedLogs: "Ayrıntılı hata ayıklama günlükleri"
+        showLogButton: "Günlük çıktısını göster"
+      modSettings:
+        title: "Mod ayarları"
+        description: "Mod ayarlarınızı kolayca dışa aktarın veya paylaşın."
+        loadButton: "Yükle"
+        loadFormattedButton: "Load formatted"
+        saveButton: "Kaydet"
+        invalidData: "Geçersiz JSON verileri"
+      customList:
+        titleInclusion: "Özel işlem dahil etme listesi"
+        descriptionInclusion: "Modun hedefleyeceği ek yürütülebilir dosya adlarının/yollarının özel bir listesi. Liste, hangi işlemlerin hedefleneceğini belirlemek için mod yazarının tanımladığı dahil etme listesine eklenir. Her işlem için, yürütülebilir dosya yolu dahil etme girişlerinden biriyle eşleşirse ve herhangi bir hariç tutma girişiyle eşleşmezse mod yüklenir."
+        titleExclusion: "Özel işlem hariç tutma listesi"
+        descriptionExclusion: "Modun hedefleme dışında bırakacağı ek yürütülebilir dosya adlarının/yollarının özel bir listesi. Liste, hangi işlemlerin hedefleneceğini belirlemek için mod yazarının tanımladığı dışlama listesine eklenir. Her işlem için, yürütülebilir dosya yolu dahil etme girişlerinden biriyle eşleşirse ve herhangi bir hariç tutma girişiyle eşleşmezse mod yüklenir."
+        processListPlaceholder: "İşlem adları/yolları, örneğin her satıra bir tane:"
+        saveButton: "Kaydet"
+    changes:
+      title: "Son Sürüm Değişiklikleri"
+      noData: "Yüklü sürüm, en son sürümle aynı."
+      splitView: "Bölünmüş görünüm"
+      expandLines_one: "Bir gizli satırı genişlet"
+      expandLines_other: "{{count}} gizli satırı genişlet"
+  home:
+    browse: "Modlara Göz At"
+    installedMods:
+      title: "Yüklü Modlar"
+      noMods: "Hiçbir mod yüklü değil"
+    featuredMods:
+      title: "Öne Çıkan Modlar"
+      noMods: "Henüz yüklenmemiş öne çıkan mod yok"
+      explore: "Diğer Modları Keşfet"
+  explore:
+    search:
+      placeholder: "Modları ara..."
+      popularAndTopRated: "Popüler ve en beğenilenler"
+      popular: "Popüler"
+      topRated: "En beğenilenler"
+      newest: "Yeniler"
+      lastUpdated: "Son güncellenenler"
+      alphabeticalOrder: "Alfabetik sıra"
+  settings:
+    language:
+      title: "Dil"
+      description: "Windhawk için tercih ettiğiniz görüntüleme dilini seçin."
+      contribute: "<0>Yeni bir çeviriye katkıda bulunun</0>."
+      credits: "Türkçe çeviri: <0>Fatih Fırıncı</0>."
+      creditsLink: "mailto:john.smith@example.com"
```

Comment:
Please replace with your email or website, or, if you prefer not to have a link, remove the `<0>...</0>` tag from the `credits` string.